doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -372,13 +356,9 @@ temperature<=20
   matches entities which don't have a type.</li>
 </ul>
 
-</td></tr>
-<tr><td style="padding-right: 40px; width: 140px;">geometry</td><td>
-
-<p>Defines a geografical area so only the entities located in that area matches the query. It is composed of a tokens list
-separated by <code>;</code>. The first token is the shape of the geometry, the rest of the tokens (if any) depends
-on the shape. In particular, the following geometries can be defined so far:</p>
-
+<p><code>geometry</code> Defines a geografical area so only the entities located in that area matches the query.
```

Comment:
I understand you have started by removing the required/optional parameter tables, moving them to the "right panel" at apiary. However, doing so you are lossing information, as the required/optional nature of the attribute cannot be inferred from the "righ panel", as far as I checked.